server/util/reingest-utils.ts

```diff
@@ -143,10 +143,34 @@ export async function reingest(queryInput: AWS.DynamoDB.QueryInput) {
   };
 }
 
-export async function reingestByDocumentId(documentId: string) {
-  return reingest(makeDocumentIdQuery(documentId));
+export async function reingestBroadcast(queryInput: AWS.DynamoDB.QueryInput) {
+  const tables = config.dynamo.symptomTable.split(',');
+  let lambdaInvocationCount = 0;
+  let recordsIngested = 0;
+
+  for (const table of tables) {
+    const result = await reingest({
+      ...queryInput,
+      TableName: table,
+    });
+    lambdaInvocationCount += result.lambdaInvocationCount;
+    recordsIngested += result.recordsIngested;
+
+    // Avoid additional querying since we've already hit found/processed the record.
+    if (lambdaInvocationCount || recordsIngested) {
+      continue;
+    }
```

Comment:
This chunk doesn't seem to be doing anything. Is that `continue` mean to be a `break`?